### DISCUSSION TITLE
Stuck with start

### DISCUSSION THREAD

**Discussion by abdulqadir7780** [lesstif/php-jira-rest-client#448] [General]
Hello everyone, 
I am new to the PHP library can anyone help me with how to set up this library with an existing PHP project. 

I have created a new file using the starter code given in the readme file. but not working give me an error "Fatal error: Uncaught Error: Class 'JiraClient\Credential' not found in C:\wamp64\www\jira-test\src\JiraClient.php on line 34"
thanks

**Comment by lesstif** [comment]
Hi @abdulqadir7780 
Credential class does not exist in this library. 
Could you show me the sample code, please?
